components/dashboard/src/admin/TeamsSearch.tsx

```diff
@@ -0,0 +1,118 @@
+            setCurrentTeam(undefined);
+        }
+    }, [location]);
+
+    if (!user || !user?.rolesOrPermissions?.includes('admin')) {
+        return <Redirect to="/"/>
+    }
+
+    if (currentTeam) {
+        return <TeamDetail team={currentTeam} />;
+    }
+
+    const search = async () => {
+        setSearching(true);
+        try {
+            const result = await getGitpodService().server.adminGetTeams({
+                searchTerm,
+                limit: 100,
+                orderBy: 'creationTime',
+                offset: 0,
+                orderDir: "desc"
+            })
+            setSearchResult(result);
+        } finally {
+            setSearching(false);
+        }
+    }
+    return <>
+        <div className="pt-8 flex">
+            <div className="flex justify-between w-full">
+                <div className="flex">
+                    <div className="py-4">
+                        <svg className={searching ? 'animate-spin' : ''} width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path fillRule="evenodd" clipRule="evenodd" d="M6 2a4 4 0 100 8 4 4 0 000-8zM0 6a6 6 0 1110.89 3.477l4.817 4.816a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 010 6z" fill="#A8A29E" />
+                        </svg>
+                    </div>
+                    <input type="search" placeholder="Search Teams" onKeyDown={(k) => k.key === 'Enter' && search()} onChange={(v) => { setSearchTerm(v.target.value) }} />
+                </div>
+                <button disabled={searching} onClick={search}>Search</button>
+            </div>
+        </div>
+        <div className="flex flex-col space-y-2">
+            <div className="px-6 py-3 flex justify-between text-sm text-gray-400 border-t border-b border-gray-200 dark:border-gray-800 mb-2">
+                <div className="w-7/12">Name</div>
+                <div className="w-5/12 flex items-center"><span>Created</span>
+                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" className="h-4 w-4" viewBox="0 0 16 16"><path fill="#A8A29E" fill-rule="evenodd" d="M13.366 8.234a.8.8 0 010 1.132l-4.8 4.8a.8.8 0 01-1.132 0l-4.8-4.8a.8.8 0 111.132-1.132L7.2 11.67V2.4a.8.8 0 111.6 0v9.269l3.434-3.435a.8.8 0 011.132 0z" clip-rule="evenodd" /></svg>
+                </div>
+
+
+            </div>
+            {searchResult.rows.map(team => <TeamResultItem team={team} />)}
+        </div>
+    </>
+
+function TeamResultItem(props: { team: Team }) {
+    return (
+        <Link key={'pr-' + props.team.name} to={'/admin/teams/' + props.team.id} data-analytics='{"button_type":"sidebar_menu"}'>
+            <div className="rounded-xl whitespace-nowrap flex py-6 px-6 w-full justify-between hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light group">
+                <div className="flex flex-col w-7/12 truncate">
+                    <div className="font-medium text-gray-800 dark:text-gray-100 truncate">{props.team.name}
```

Comment:
**question**: Could we include the members count below the team name here as described in the specs[[1](https://github.com/gitpod-io/gitpod/issues/8060#issuecomment-1047869502)]? Is this an expensive operation or did we decide to drop it?

Asking as :a: this can provide a visual cue when searching teams and :b: showing members below the team name is a pattern we already use in the team scope selector (dropdown) in the top navigation as well as the team selection on the onboarding wizard.

| Team scope selector (dropdown) | Team selection on the onboarding wizard |
|-|-|
| <img width="352" alt="Screenshot 2022-02-24 at 12 20 19 PM" src="https://user-images.githubusercontent.com/120486/155505721-e59c2e4f-e17e-4f1d-aa93-ac8b82c68b91.png"> | <img width="392" alt="Screenshot 2022-02-24 at 12 20 45 PM" src="https://user-images.githubusercontent.com/120486/155505722-6ea3f89b-c4f5-48b2-8a12-6b5f5d579f6a.png"> |

Reply:
I opted to not do it here because since it's an extra call and I wanted to make sure the search result gets shown as fast as possible. In the top navigation, the dropdown isn't actually immediately shown - only when we do click on it, so there's time for it to load. I'm okay with including this if it's necessary for this iteration. Let me know @gtsiolis @jldec

Reply:
@laushinka If we are not paginating this list, which I think this is the case here (right?), let's skip the members count.

Reply:
I'm trying this now, since I could be optimizing early with my above thought. Coming soon!

Reply:
I wrote this comment before the page loaded with your new comment 🤣

Reply:
@gtsiolis I ended up pushing a commit with the members count. Let me know what you think once the preview env is built - I didn't rebase it with the previous one so it's easily revertible in case you'd like me to get rid of it 🤗

Reply:
See https://github.com/gitpod-io/gitpod/pull/8402#discussion_r814045854.